src/index.js

```diff
@@ -311,11 +478,11 @@ IdTokenVerifier.prototype.validateAccessToken = function(
   if (this.expectedAlg !== alg) {
     return cb(
       new error.TokenValidationError(
-        'Algorithm ' +
+        'Signature algorithm of "' +
```

Comment:
This is for the method `validateAccessToken` which is out of scope for this PR. I guess the change is welcome anyways because provides a consistent error message for both token validations when checking the algorithm.